Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -159,11 +159,28 @@ - (void)screen:(SEGScreenPayload *)payload
 
 - (void)group:(SEGGroupPayload *)payload
 {
-    NSString *groupId = payload.groupId;
-    if (groupId) {
-        [self.amplitude setGroup:@"[Segment] Group" groupName:groupId];
-        SEGLog(@"[Amplitude setGroup:@'[Segment] Group' groupName:%@]", groupId);
+    NSString *groupTypeTrait;
+    if (self.settings[@"groupTypeTrait"]) {
```

Comment:
I don't understand why this is an if check.

What's the difference between these two pieces of code:

```
NSString *groupTypeTrait = self.settings[@"groupTypeTrait"];
```
```
NSString *groupTypeTrait;
if (self.settings[@"groupTypeTrait"]) {
  groupTypeTrait = self.settings[@"groupTypeTrait"];
}
```

They seem functionally equivalent to me.